docs/source/architecture/sub_stakes.rst

```diff
@@ -0,0 +1,276 @@
+Sub-stakes
+==========
+
+A staker may extend the unlock period for any number of portions of their total stake. This divides the stake into smaller parts, each with a unique unlock date in the future. Stakers may also acquire and lock new tokens. The total stake is represented as the sum of all the different sub-stakes active in a given cycle (new cycle every 24h), which includes locked sub-stakes, and any sub-stakes that have passed their unlock date, and can be freely withdrawn. Each sub-stake has a beginning and duration (lock time). When a staker confirms activity each day and winding down is enabled, the remaining lock time for relevant sub-stakes is reduced.
```

Comment:
Should we use consistent terminology? Here we use `cycle` - is that the same thing as a `period`?

Reply:
Changed to `period`
✔️